internal/config.go

```diff
@@ -104,13 +104,14 @@ const (
 	OplogReplayOplogApplicationMode = "OPLOG_REPLAY_OPLOG_APPLICATION_MODE"
 	OplogReplayIgnoreErrorCodes     = "OPLOG_REPLAY_IGNORE_ERROR_CODES"
 
-	MysqlDatasourceNameSetting = "WALG_MYSQL_DATASOURCE_NAME"
-	MysqlSslCaSetting          = "WALG_MYSQL_SSL_CA"
-	MysqlBinlogReplayCmd       = "WALG_MYSQL_BINLOG_REPLAY_COMMAND"
-	MysqlBinlogDstSetting      = "WALG_MYSQL_BINLOG_DST"
-	MysqlBackupPrepareCmd      = "WALG_MYSQL_BACKUP_PREPARE_COMMAND"
-	MysqlTakeBinlogsFromMaster = "WALG_MYSQL_TAKE_BINLOGS_FROM_MASTER"
-	MysqlCheckGTIDs            = "WALG_MYSQL_CHECK_GTIDS"
+	MysqlDatasourceNameSetting      = "WALG_MYSQL_DATASOURCE_NAME"
+	MysqlSslCaSetting               = "WALG_MYSQL_SSL_CA"
+	MysqlBinlogReadFromRemoteServer = "WALG_MYSQL_BINLOG_READ_FROM_REMOTE_SERVER"
```

Comment:
Yeah the main reason for `WALG_MYSQL_READ_FROM_REMOTE_SERVER` was just so it would share the same name as the corresponding `--read-from-remote-server` option for `mysqlbinlog`.